tests/test_tab.py

```diff
@@ -0,0 +1,67 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import Tab, Button
+from tests.utils import WindowTestCase
+
+
+class TabTest(WindowTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tab = Tab()
+
+    def test_can_append_pages_to_tab(self):
+        """Tests that we can append pages to tab."""
```

Comment:
Ah ok.. Yes, sometimes the result of one test depends on other factors and that's ok! 
A good and short name for this test could be "test_append_pages" and a description could be something like "Pages can be appended to a tab."